### PR TITLE
Add Columndefs support and type conversion for several types.

### PR DESCRIPTION
**Pull request by brianv0**
Updates to reflect dax_ prefix changes included.

### CODE REVIEW

File: python/lsst/dax/dbserv/dbREST_v0.py [hunk 1]
```diff
@@ -60,11 +63,25 @@ def getQuery():
        If sql is passed, it runs a given query.'''
     if 'sql' in request.args:
         sql = request.args.get('sql').encode('utf8')
+        print sql
```
Comment (andy-slac) [inline python/lsst/dax/dbserv/dbREST_v0.py]: `log.debug()`?

File: python/lsst/dax/dbserv/dbREST_v0.py [hunk 2]
```diff
@@ -81,3 +98,48 @@ def _response(response, status_code):
...
+        @param name: Column name
+        @param flags: Flags from MySQLdb.constants.FLAGS
+        @param value: An example value type to help with inferring how to convert
+        """
+        self.name = name
+        self.converter = None
+        if type(value) == float:
+            self.type = "float"
+        elif type(value) == long:
+            self.type = "long"
```
Comment (andy-slac) [inline python/lsst/dax/dbserv/dbREST_v0.py]: This is not Python3-proof, `long` does not exist there. Not sure what to do about it though.

File: python/lsst/dax/dbserv/dbREST_v0.py [hunk 3]
```diff
@@ -81,3 +98,48 @@ def _response(response, status_code):
...
+        elif type(value) == date:
+            self.type = "date"
+            self.converter = lambda x: x.isoformat()
+        elif flags & BINARY and type(value) not in (date, datetime, time, timedelta):
+            # Note: It seems date, datetime, etc... show up with the BINARY flag.
+            self.type = "binary"
+            self.converter = b64encode
+        elif type(value) == str:
+            self.type = "string"
+        else:
```
Comment (andy-slac) [inline python/lsst/dax/dbserv/dbREST_v0.py]: What if value is None (NULL) for some column(s) in the first row?

File: python/lsst/dax/dbserv/dbREST_v0.py (hunk 2 above)
Comment (PaulPrice) [inline python/lsst/dax/dbserv/dbREST_v0.py]: Why not `isinstance(value, long)`?

And you could use a `dict` instead of a lot of `if..elif`s.

**Pull request merged**